src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/SpatialCoordinateSystemManager.cs

```diff
@@ -178,7 +205,7 @@ public Task<bool> RunRemoteLocalizationAsync(SocketEndpoint socketEndpoint, Guid
             if (remoteLocalizationSessions.TryGetValue(socketEndpoint, out var currentCompletionSource))
             {
                 DebugLog("Canceling existing remote localization session: {socketEndpoint.Address}");
-                currentCompletionSource.TrySetCanceled();
+                currentCompletionSource.TrySetResult(false);

```

Comment:
>TrySetResult [](start = 40, length = 12)

Why not cancel here?

Reply:
No preference. I guess I thought this would be called from a user action, but I will just revert.